### DISCUSSION TITLE
為什麼用WireGuard有時候會慢？

### DISCUSSION THREAD

**Discussion by okudayukiko** [v2fly/v2ray-core#1074] [General]
用WireGuard，可以看YouTube 2160P，但是看niconico畫質低，看某些直播（如www.tdm.com.mo）也會卡住。
用Shadowsocks就沒這問題。

**Comment by okudayukiko** [comment]
以我的英國VPS為例，改用英國DNS，問題解決。
用8.8.8.8，以及8.8.8.8 + DNS快取，都有這問題。

**Comment by ghost** [comment]
It seems that this discussion is not related to V2Ray.
